pyproject.toml

```diff
@@ -33,7 +33,9 @@ dev = [
     "coverage[toml]>=7.8.0",
     "mujoco-warp>=3.2.7",
     "usd-core>=25.5 ; platform_machine != 'aarch64'",
+    "pycollada>=0.9",
     "pyglet>=2.1.6",
+    "torch>=2.7.0",
```

Comment:
Can you remove the torch dependency? I think this will cause issues when users need torch with CUDA etc. but pip selects the wrong version.